mig/shared/cgiscriptstub.py

```diff
@@ -98,6 +98,13 @@ def finish_cgi_script(configuration, backend, output_format, ret_code, ret_msg,
         start_entry['headers'] = default_headers
     headers = start_entry['headers']
 
+    if ret_code == SIZE_LIMIT_ERROR[0]:
```

Comment:
[Comment](https://github.com/ucphhpc/migrid-sync/pull/139/files#r1806447212) also applies here.

Reply:
removed in latest version.